### PR TITLE
Scala 3.0.0 for 1.x series (CE3)

### PR DESCRIPTION
**Pull request by jatcwang**

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -80,8 +79,8 @@ lazy val commonSettings =
 
     // MUnit
     libraryDependencies ++= Seq(
-      "org.typelevel"     %% "scalacheck-effect-munit" % "1.0.1"  % Test,
-      "org.typelevel"     %% "munit-cats-effect-2"     % "1.0.2" % Test,
+      "org.typelevel"     %% "scalacheck-effect-munit" % "1.0.2"  % Test,
+      "org.typelevel"     %% "munit-cats-effect-2"     % "1.0.3" % Test,
```
Comment (jatcwang) [inline build.sbt:83]: @tpolecat should this be munit-cats-effect-3 instead? :thinking:

Comment (tpolecat) [reply build.sbt:83]: Yes, curious that it works at all.

Comment (tpolecat) [reply build.sbt:83]: We're good to go if you can fix this.

**Comment by ahjohannessen** [comment]
:shipit:

**Pull request merged**